src/test/java/nextstep/subway/unit/SectionsTest.java

```diff
@@ -164,4 +165,16 @@ void deleteSectionFail() {
         assertThatThrownBy(() -> 신분당선.deleteSection(양재역))
                 .isInstanceOf(IllegalArgumentException.class);
     }
-}
\ No newline at end of file
+
+    @DisplayName("sections의 전체 거리를 계산한다.")
+    @Test
+    void totalDistance() {
+        // given
+        Station 판교역 = new Station("판교역");
+        신분당선.addSection(양재역, 판교역, 6);
+        Sections sections = new Sections(신분당선.getSections());
+
+        // when & then
+        assertThat(sections.totalDistance()).isEqualTo(16);
+    }
```

Comment:
테스트가 좀 더 깨지지 않도록 만들기 위한 팁은 이 테스트에서 `16`과 같은 상수 값을 직접 사전에 연산해서 유추해서 작성하기보다는,  어째서 16이 나오는지도 알 수 있게 작성해주시는게 좋습니다. 

```suggestion
    void totalDistance() {
        // given
        int newDistance = 6;
        Station 판교역 = new Station("판교역");
        신분당선.addSection(양재역, 판교역, newDistance);
        Sections sections = new Sections(신분당선.getSections());

        // when & then
        assertThat(sections.totalDistance()).isEqualTo(distance + newDistance);
    }
```

이런식으로 말이죠.